### PR TITLE
Feature/#51 update systems

### PR DESCRIPTION
**Pull request by kgingstad**

### CODE REVIEW

File: arxivdigest/frontend/database/general.py [hunk 1]
```diff
@@ -92,16 +92,15 @@ def insertUser(user):
     return id
 
 
-def insertSystem(system):
+def insertSystem(system_name, user_id):
     """Inserts a new system into the database, name will be used as Name for the system,
     and using uuid a random API-key is generated. Returns None if successfull and an error if not."""
```
Comment (kbalog) [inline arxivdigest/frontend/database/general.py]: what is uuid?

File: arxivdigest/frontend/static/javascript/admin.js [hunk 2]
```diff
@@ -100,8 +100,8 @@ function generateSystemTableHtml(systems) {
     for (const system of systems) {
         html += "<td>" + system.system_ID + "</td>";
         html += "<td>" + system.system_name + "</td>";
-        html += "<td>" + system.contact_name + "</td>";
-        html += "<td>" + system.organization_name + "</td>";
+        html += "<td>" + system.firstname + " " + system.lastname + "</td>";
+        html += "<td>" + system.company + "</td>";
```
Comment (kbalog) [inline arxivdigest/frontend/static/javascript/admin.js]: company => organization

File: arxivdigest/frontend/templates/admin.html [hunk 3]
```diff
@@ -29,7 +29,7 @@ <h3 class="leftText">List of systems:</h3>
                                 <th>ID</th>
                                 <th>Name</th>
                                 <th>Contact name</th>
-                                <th>Organization name</th>
+                                <th>Company name</th>
```
Comment (kbalog) [inline arxivdigest/frontend/templates/admin.html]: Company => Organization

File: arxivdigest/frontend/database/general.py (hunk 1 above)
Comment (kgingstad) [reply arxivdigest/frontend/database/general.py]: Universally unique identifiers. Used to get unique api keys

**Pull request merged**